lib/cdo/delete_accounts_helper.rb

```diff
@@ -308,6 +308,13 @@ def purge_teacher_feedbacks(user_id)
     @log.puts "Cleared #{as_student_count} TeacherFeedback" if as_student_count > 0
   end
 
+  def delete_ai_tutor_interactions(user_id)
+    chat_messages_to_delete = AiTutorInteraction.where(user_id: user_id)
+    count = chat_messages_to_delete.count
+    chat_messages_to_delete.destroy_all
```

Comment:
It's safer to perform such operations `in batches`
```suggestion
    chat_messages_to_delete.in_batches.destroy_all
```